robottelo/ui/rhai.py

```diff
@@ -16,3 +17,19 @@ def view_registered_systems(self):
             locators['insights.registered_systems']
         ).text
         return result
+
+    def unregister_system_from_inventory(self, system_name, confirm=True):
+        """Unregister system from inventory"""
+        Navigator(self.browser).go_to_insights_inventory()
```

Comment:
any reason to not use our default approach with overloaded method?
e.g.
```
def navigate_to_entity(self):
        """Navigate to Compute Profile entity page"""
        Navigator(self.browser).go_to_compute_profiles()
```

Reply:
This is not a pure entity, this a class for many different pages

Reply:
maybe we create class per each entity
like:
module - rhai
but
classes - inventory and etc